docs/data/theme/external-data/_data.yaml

```diff
@@ -1,5 +1,5 @@
 title: External data
-description: External data products are not produced by DEA; they are produced by external providers such as a different program of Geoscience Australia or a different organisation entirely. These external data products may assist with the analysis of DEA data.
+description: External data products are not produced by DEA; they are produced by external providers such as a different program of Geoscience Australia or a different organisation entirely. These external data products may assist with the analysis of DEA data. This data hasn't been altered except to make it compatible with DEA's systems.
```

Comment:
Slight wording update - might still be a bit wordy but I think putting the "purpose" up front helps.

> **These external data products may assist with the analysis of DEA data.** External data products are not produced by DEA; they are produced by external providers such as a different program of Geoscience Australia or a different organisation entirely. This data **has not** been altered except to make it compatible with DEA's systems.

Reply:
I actually had the same idea - I wrote this in my latest edit:

> This external data is provided because it may assist with the analysis of DEA data. These data products are produced by external parties, and are not DEA products. DEA provides this data without modifications except where needed to make it compatible with our systems.

Reply:
Sounds great!